.travis.yml

```diff
@@ -16,8 +16,7 @@ install:
 - export PATH=$PWD/bin:$PATH
 - ./ci/install.sh
 - pip install -r dev-requirements.txt
-- npm install
-- npm run webpack
+- python3 setup.py install
```

Comment:
`setup.py install` should ~never be used to install a Python package. It should always be `pip install .`

Reply:
Done. Do you have pointers to why?